images/bun-node/Dockerfile

```
@@ -1,57 +1,38 @@
+ARG OXFMT_VERSION="0.56.0"
+
+# 3. New tool ARGs for this specific derivative
+# renovate: datasource=node-version depName=node
+ARG NODE_VERSION="22.11.0"
+
 ARG USERNAME=root
 
-# 1. Install packages, global NPM tools, generate versions, and install Antigravity
+# 4. Elevate to root, install Node/NPM, and drop privileges
+USER root
 RUN --mount=type=cache,target=/var/cache/apk \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)